sdk/typescript/src/rpc/client.ts

```diff
@@ -45,8 +46,14 @@ export class JsonRpcClient {
         try {
           let res: Response = await fetch(url, options);
           const text = await res.text();
+          const result = JSON.stringify(LosslessJSON.parse(text, (key : string, value : any) => {
+            if (key === "balance") return value.toString(); 
+            if (value.isLosslessNumber) return value.valueOf();
```

Comment:
This will throw an exception when value == null or value is greater than MAX_SAFE_INT